plugins/alignments/src/PileupRenderer/PileupRenderer.ts

```diff
@@ -370,15 +372,17 @@ export default class PileupRenderer extends BoxRendererType {
                   Math.abs(softClipLeftPx - softClipRightPx),
                 )
 
-                // Black accounts for IUPAC ambiguity code bases such as N that show in soft clipping
-                ctx.fillStyle = lighten(colorForBase[base] || '#000000', 0.3)
+                // Black accounts for IUPAC ambiguity code bases such as N that
+                // show in soft clipping
+                const baseColor = colorForBase[base] || '#000000'
+                ctx.fillStyle = baseColor
```

Comment:
It looks like the lightening of the color of the softclipped bases was removed. We can use `theme.palette.bases.<base>.light` to keep that behavior.

Reply:
Forgot to mention but I propose removing the lightening. We now render the clip indicator in all cases, even when soft clip is showing, so that it is clear when bases are being drawn beyond that)